### PR TITLE
201811: Upgrade redis-server from 3.2.4 to 3.2.13

### PR DESCRIPTION
**Pull request by renukamanavalan**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/redis/Makefile [hunk 1]
```diff
@@ -2,17 +2,17 @@ SHELL = /bin/bash
...
+    wget -O redis_$(REDIS_VERSION_FULL).dsc -N "https://nam06.safelinks.protection.outlook.com/?url=https:%2F%2Fsonicstorage.blob.core.windows.net%2Fpackages%2Fredis%2Fredis_3.2.13-1~bpo8%2B1.dsc%3Fsv%3D2015-04-05%26sr%3Db%26sig%3DmQ502FHWRiOFb9cjEUcK0r1gXAc3bBv0fmVEWtroy5c%253D%26se%3D2033-05-19T16%253A43%253A29Z%26sp%3Dr&data=02%7C01%7CRenuka.Manavalan%40microsoft.com%7C0b8c3585f5614ad0ae3808d7360e2296%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637037306647651889&sdata=bFx8z%2FglIZ4j%2F4H8NzHeoju%2BbVK0BoGfkfX%2FVtTbykk%3D&reserved=0"
+    wget -O redis_$(REDIS_VERSION_FULL).debian.tar.xz -N "https://nam06.safelinks.protection.outlook.com/?url=https:%2F%2Fsonicstorage.blob.core.windows.net%2Fpackages%2Fredis%2Fredis_3.2.13-1~bpo8%2B1.debian.tar.xz%3Fsv%3D2015-04-05%26sr%3Db%26sig%3DiboHl5%252BV6zUcS5%252B8CaNnE9A0qLa7H%252BU8P1H0eofWjig%253D%26se%3D2033-05-19T16%253A43%253A00Z%26sp%3Dr&data=02%7C01%7CRenuka.Manavalan%40microsoft.com%7C0b8c3585f5614ad0ae3808d7360e2296%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637037306647651889&sdata=EPk1mIb67YKKLsMLFBLeRQxLC981mn0RE0SHh2weVSA%3D&reserved=0"
```
Comment (jleveque) [inline src/redis/Makefile]: Replace spaces with tab on lines 13-15.

Comment (renukamanavalan) [reply src/redis/Makefile]: Thanks Joe! Fixed.

File: src/redis/Makefile [hunk 2]
```diff
@@ -2,17 +2,17 @@ SHELL = /bin/bash
...
+	wget -O redis_$(REDIS_VERSION_FULL).dsc -N "https://nam06.safelinks.protection.outlook.com/?url=https:%2F%2Fsonicstorage.blob.core.windows.net%2Fpackages%2Fredis%2Fredis_3.2.13-1~bpo8%2B1.dsc%3Fsv%3D2015-04-05%26sr%3Db%26sig%3DmQ502FHWRiOFb9cjEUcK0r1gXAc3bBv0fmVEWtroy5c%253D%26se%3D2033-05-19T16%253A43%253A29Z%26sp%3Dr&data=02%7C01%7CRenuka.Manavalan%40microsoft.com%7C0b8c3585f5614ad0ae3808d7360e2296%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637037306647651889&sdata=bFx8z%2FglIZ4j%2F4H8NzHeoju%2BbVK0BoGfkfX%2FVtTbykk%3D&reserved=0"
+	wget -O redis_$(REDIS_VERSION_FULL).debian.tar.xz -N "https://nam06.safelinks.protection.outlook.com/?url=https:%2F%2Fsonicstorage.blob.core.windows.net%2Fpackages%2Fredis%2Fredis_3.2.13-1~bpo8%2B1.debian.tar.xz%3Fsv%3D2015-04-05%26sr%3Db%26sig%3DiboHl5%252BV6zUcS5%252B8CaNnE9A0qLa7H%252BU8P1H0eofWjig%253D%26se%3D2033-05-19T16%253A43%253A00Z%26sp%3Dr&data=02%7C01%7CRenuka.Manavalan%40microsoft.com%7C0b8c3585f5614ad0ae3808d7360e2296%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637037306647651889&sdata=EPk1mIb67YKKLsMLFBLeRQxLC981mn0RE0SHh2weVSA%3D&reserved=0"
 	dpkg-source -x redis_$(REDIS_VERSION_FULL).dsc
```
Comment (yxieca) [inline src/redis/Makefile]: Can you remove the nam06 prefix. Just use the azure storage links?

Comment (renukamanavalan) [reply src/redis/Makefile]: Just noticed. Will update soon.

Comment (yxieca) [reply src/redis/Makefile]: It is best to use (REDIS_VERSION) and (REDIS_VERSION_FULL) at all places (URL and TARGET), or none.

Comment (renukamanavalan) [reply src/redis/Makefile]: Fixed that too. Testing locally

**Comment by lguohan** [comment]
looks like there is build failure.

**Comment by lguohan** [comment]
https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/buildimage-mlnx-all-released-pr/120/console

**Comment by renukamanavalan** [comment]
I will update upon local testing.

File: src/redis/Makefile (hunk 2 above)
Comment (lguohan) [reply src/redis/Makefile]: where do you get the debian patches for 3.2.13? is it from debian official repo?

Comment (lguohan) [reply src/redis/Makefile]: LGTM

**Pull request merged**